scripts/dmodels_main.dsc

```diff
@@ -134,9 +134,9 @@ dmodels_config:
     # You can set the resource pack path to a custom path if you want.
     # Note that the default Denizen config requires this path start under "data/"
     resource_pack_path: data/dmodels/res_pack
-    # During the loading process the scale is shrunken down exactly 2.2 for items to allow bigger models in block bench so this brings it back up to default scale in-game (this is an estimate)
-    default_scale: 2.2
+    # During the loading process the scale is shrunken down for items to allow bigger models in block bench so this brings it back up to default scale in-game (this is an estimate)
+    default_scale: 4.8
```

Comment:
Why _is_ this so different from the, yknow, value in the loading code? Which was indeed 2.2 before, but you changed to 0.25/4 in a prior PR?

Reply:
This is just to scale the model up after scaling it down in the loader to allow bigger models that's it. As to why I changed it to 4.8 when combining the spawning task with reset_model_position the global scale got duplicated I didn't notice this before but I fixed it.

Reply:
Oh, and the previous value made the model small after making the fix.